sdk/communication/azure-communication-common-perf/src/main/java/com.azure.communication.common.perf/HmacAuthenticationPolicyTest.java

```diff
@@ -0,0 +1,94 @@
+package com.azure.communication.common.perf;
+
+import com.azure.communication.common.implementation.HmacAuthenticationPolicy;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.*;
+import com.azure.core.test.http.MockHttpResponse;
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.perf.test.core.PerfStressTest;
+import reactor.core.publisher.Mono;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class HmacAuthenticationPolicyTest extends PerfStressTest<PerfStressOptions> {
+
+    public class CustomPair {
+        String signature;
+        int count;
```

Comment:
I am wondering if  `count` shouldn't be rather `AtomicInteger` because you are incrementing it from multiple threads.

Reply:
yeah, you're right, will fix it